package.json

```diff
@@ -33,7 +33,7 @@
     "changelog": "lerna-changelog",
     "lint": "npm-run-all --continue-on-error --aggregate-output --parallel lint:*",
     "lint:docs": "markdownlint '**/*.md'",
-    "lint:js": "eslint . --cache",
+    "lint:js": "eslint . --cache --fix",
```

Comment:
Fixing should be done as part of a separate script:

```
"lint:js:fix": "yarn lint:js --fix"
```

Since sometimes we just want to check for issues and not fix them (during CI for example).